### ISSUE TITLE
Just tried to use this on a new Drupal project and get a memory issue

### ISSUE THREAD

**Issue by chris-archer-whisky** [guix77/phpcs-drupal-action#1]
Just tried to use this on a new Drupal project and get a memory issue at the Run guix77/phpcs-drupal-action@v1.0.0 step.

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 8388616 bytes) in /root/.composer/vendor/squizlabs/php_codesniffer/src/Tokenizers/CSS.php on line 251

**Comment by mnico** [comment]
Same here

```
Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in /root/.composer/vendor/squizlabs/php_codesniffer/src/Tokenizers/PHP.php on line 1524
```

Maybe in the Docker image created from here https://github.com/guix77/phpcs-drupal-action needs define more memory limit?

Regards

**Comment by mnico** [comment]
I have proposed the possibility of defining the amount of memory limit to use.

**Comment by chadedge-cbsi** [comment]
bump - I would love to see that addition.

**Comment by chris-archer-whisky** [comment]
I got around the issue without using this, here is my code:

```

name: PHPUnit Tests

on: push

jobs:
  phpspec:
    name: Setup and run the PHPUnit tests
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: '7.4'
          coverage: none
          tools: composer:v2, cs2pr
      - name: Run composer install
        run: composer install
      - name: Show phpUnit tests results in PR
        continue-on-error: true
        run: |
          vendor/bin/phpunit --debug --verbose web/themes/custom | cs2pr
          vendor/bin/phpunit --debug --verbose web/modules/custom | cs2pr
      - name: Run phpUnit tests
        run: |
          vendor/bin/phpunit --debug --verbose web/themes/custom
          vendor/bin/phpunit --debug --verbose web/modules/custom
```

**Comment by guix77** [comment]
It's now included in v1.0.1. Thank you guys!
